client/app/account/user/credentials/user-credentials.html

```diff
@@ -0,0 +1,103 @@
+<div class="module-useraccount-sections-credentials-container"
+     data-ng-controller="UserAccount.controllers.credentials as $ctrl">
+
+    <div class="page-header">
+        <h1 data-i18n-static="user_credentials_title"></h1>
+    </div>
+
+    <div class="tab-content">
+
+        <div data-ovh-alert="userCredentials"></div>
+
+        <!-- CREDENTIALS EXPLAIN -->
+        <div class="alert alert-info"
```

Comment:
Also, you can refer to our components library: http://master.ui-kit.ovh/#!/oui-angular/message

For instance: 
```html
<oui-message data-type="info">
  <span data-i18n-static="user_credentials_info"></span>
</oui-message>
```